tests/resources/spellcheck/stopwords.txt

```diff
@@ -230,9 +232,11 @@ PPID
 prct
 Primera
 Procurve
+Proxmox
 proto
 --proxyurl
 psu
+pve
```

Comment:
`pve` does not look like a valid word, I'd prefer it to be quoted as code (`C<pve>`) in the POD